docs/contributor-guide/how-to/rule.md

```diff
@@ -65,24 +67,10 @@ If you are working in `sonarwhal`’s main repository, one of the easiest ways
 to get started is to use `sonarwhal`’s CLI, which helps to generate the template
 files and insert them at the right location.
 
-First you need to install the CLI:
+After cloning your fork of `sonarwhal` do the following:
```

Comment:
Why cloning?

Reply:
We are talking about the core rules here, so the user needs to clone the main project (or their fork) in their machine.